libbeat/docs/version.asciidoc

```diff
@@ -1,4 +1,4 @@
-:stack-version: 5.5.0
-:doc-branch: 5.5
+:stack-version: 5.6.0
```

Comment:
@monicasarbu did you run set-version for this?
@tsg Kind of concerned that we miss that somewhere else.